sdk/cosmos/azure-cosmos-encryption/README.md

```diff
@@ -0,0 +1,10 @@
+# Azure CosmosDB Client Library Encryption Extension
+
+To add client side encryption support you need to add
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-cosmos-encryption</artifactId>
+  <version>0.9.0-beta.1</version> 
+</dependency>
+```
```

Comment:
You're missing the update tags which belong before the begin and after the end of the XML tag block. Which should be
`[//]: # ({x-version-update-start;com.azure:azure-cosmos-encryption;current})`
`[//]: # ({x-version-update-end})`

Reply:
thanks done.